Makefile

```diff
@@ -163,14 +167,14 @@ fmt:
 .PHONY: help
 ### help: print this message
 help: Makefile
-	@echo "Available rules:"
+	@echo 'Available rules:'
 	@sed -n 's/^### /    /p' $< | awk 'BEGIN { FS=":" } { printf "%-22s -%s\n", $$1, $$2 }'
-	@echo ""
-	@echo "Supported environment variables:"
-	@echo "    IMG                - Image used for controller"
-	@echo "    NAMESPACE          - Namespace to use for deploying controller"
-	@echo "    TOOL               - CLI tool for interfacing with the cluster: kubectl or oc; if oc is used, deployment is tailored to OpenShift, otherwise Kubernetes"
-	@echo "    CLUSTER_IP         - For Kubernetes only, the ip address of the cluster (minikube ip)"
-	@echo "    PULL_POLICY        - Image pull policy for controller"
-	@echo "    WEBHOOK_ENABLED    - Whether webhooks should be enabled in the deployment"
-	@echo  "    ADMIN_CTX          - Kubectx entry that should be used during work with cluster. The current will be used if omitted"
+	@echo ''
+	@echo 'Supported environment variables:'
+	@echo '    IMG                - Image used for controller'
+	@echo '    NAMESPACE       http://127.0.0.1:44299/api/v1/namespaces/kubernetes-dashboard/services/http:kubernetes-dashboard:/proxy/#/overview?namespace=che-workspace-controller   - Namespace to use for deploying controller'
+	@echo '    TOOL               - CLI tool for interfacing with the cluster: kubectl or oc; if oc is used, deployment is tailored to OpenShift, otherwise Kubernetes'
+	@echo '    ROUTING_SUFFIX     - Cluster routing suffix (e.g. $$(minikube ip).nip.io, apps-crc.testing)'
```

Comment:
The issue I've had is that *not* setting it correctly initially sometimes (but not always) results in the wrong routing suffix being used, even on OpenShift. I'll create an issue if I can find a consistent reproducer.

The functionality for rewriting routingSuffix (formerly ingress.global.domain) has existed since we started working on the project but we've all been bit by it being set incorrectly at some point.